root.spec

```diff
@@ -39,7 +40,7 @@ export CXXFLAGS=-D__ROOFIT_NOBANNER
 
 cmake ../%{n}-%{realversion} \
   -G Ninja \
-  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
@mrodozov , I would suggest to build it in Release mode

Reply:
i was going to ask if the new cmake was fixing this 😅 it's also need to be changed in master

Reply:
issue with cmake was that it was not allowing us to build with `-DCMAKE_BUILD_TYPE=Debug -DLLVM_BUILD_TYPE=Release` (i.e. one with Release and other with Debug). For root master we want to build with `-DCMAKE_BUILD_TYPE=Debug`  bit for normal IBs and rootnext we want to be in `Release` mode (i.e. close to the normal IBs)